index.js

```diff
@@ -31,6 +32,22 @@ app.post("/message", (req, res) => {
   res.json(message);
 });
 
+app.post("/message/:id/reaction", (req, res) => {
+  if (!req.body.user) {
+    return res.status(400).json({ error: "Missing user" });
+  }
+  if (!req.body.reaction) {
+    return res.status(400).json({ error: "Missing reaction" });
+  }
+  const reaction = {
+    user: req.body.user,
+    reaction: req.body.reaction,
+    createdAt: new Date(),
+  };
+  reactions.push(reaction);
```

Comment:
We should check to make sure there is no same reaction already recorded by the user